.github/workflows/test.yml

```diff
@@ -41,6 +41,7 @@ jobs:
             galsim \
             scikit-learn \
             emcee \
+            fitsio \
```

Comment:
Do we need to add this to the conda package deps too?

Reply:
that is for testing the save_mixture, load_mixture for the gmix_ndim stuff.

I'm not sure anyone even uses that class, so I'm not sure if its worth having it in the deps

Reply:
We will also need it for testing the medsreaders